client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -42,13 +42,14 @@ describe( 'EditGravatar', function() {
 		};
 	} );
 
-	before( function() {
+	before( function( done ) {
```

Comment:
would be happy to discuss this. tests should never be intermittent or nondeterministic. also `done` is there for a very specific purpose (async flows) so it's not a hack to use it. I'm not sure how you were using it but you may be too hard on yourself :smile: